var/spack/repos/builtin/packages/r-aer/package.py

```diff
@@ -12,16 +12,17 @@ class RAer(RPackage):
     Applied Econometrics with R, Springer-Verlag, New York.
     ISBN 978-0-387-77316-2."""
 
-    homepage = "https://cran.r-project.org/web/packages/AER/index.html"
-    url      = "https://cran.r-project.org/src/contrib/AER_1.2-5.tar.gz"
+    homepage = "https://cloud.r-project.org/web/packages/AER/index.html"
```

Comment:
Switch to https://cloud.r-project.org/package=AER

Reply:
This is fixed with the latest commit.